### PR TITLE
Task 4 - Stats queries

### PR DESCRIPTION
**Pull request by Rxbsxn**

### CODE REVIEW

**Comment by kovson** [comment]
@Rxbsxn 
I assume specs for scopes are just not pushed :) Waiting for them.
You can merge rest of branches.

File: spec/models/movie_spec.rb [hunk 1]
```diff
@@ -17,8 +17,18 @@
...
   describe 'should belong_to genre' do
     it { should belong_to(:genre) }
   end
+
+  describe '.max_revenue' do
+    let(:movie) { create(:movie) }
+    let(:movie2) { create(:movie) }
+    
+    it 'returns movie with maximum value' do
+      movie.update_attributes(revenue: 99999999)
```
Comment (kovson) [inline spec/models/movie_spec.rb]: It is not necessary, you can assign revenue while creating an object.

**Comment by kovson** [comment]
@Rxbsxn 
One small change, otherwise good job.

File: spec/models/movie_spec.rb [hunk 2]
```diff
@@ -23,11 +23,11 @@
...
 
   describe '.max_revenue' do
-    let(:movie) { create(:movie) }
+    let(:movie) { create(:movie, revenue: 99999999) }
     let(:movie2) { create(:movie) }
     
-    it 'returns movie with maximum value' do
-      movie.update_attributes(revenue: 99999999)
+    it 'returns movie with maximum value' do    
+      movie
```
Comment (Rxbsxn) [inline spec/models/movie_spec.rb]: @kovson Why I have to reuse movie object in `it` block to make the test passing. I was trying to figure it out with using pry gem, but everything was good revenue was same like in `Movie.max_revenue` but test was falling with error about wrong variable `name`. If you have some ideas why please give me know

**Pull request merged**